### PR TITLE
imxrt1050-evk/loadable_elf_apps/defconfig: Enable FS_CMDS

### PR DESCRIPTION
**Pull request by sangam-swami**
Enabled fs commands config for loadable elf apps

Signed-off-by: Sangamanatha <sangam.swami@samsung.com>

### CODE REVIEW

File: build/configs/imxrt1050-evk/loadable_elf_apps/defconfig [hunk 1]
```diff
@@ -859,6 +875,7 @@ CONFIG_ELF_STACKSIZE=2048
 CONFIG_ELF_BUFFERSIZE=32
 CONFIG_ELF_BUFFERINCR=32
 CONFIG_ELF_EXCLUDE_SYMBOLS=y
+# CONFIG_ELF_CACHE_READ is not set
```
Comment (sunghan-chang) [inline build/configs/imxrt1050-evk/loadable_elf_apps/defconfig]: Could you enable this as well?

**Comment by seinfra** [comment]
Target : [339a335213ef4f3079488ff5d1b863407d98e6f7] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [339a335213ef4f3079488ff5d1b863407d98e6f7] - Code Rule Check (C++) OK.

File: build/configs/imxrt1050-evk/loadable_elf_apps/defconfig [hunk 2]
```diff
@@ -859,6 +875,9 @@ CONFIG_ELF_STACKSIZE=2048
 CONFIG_ELF_BUFFERSIZE=32
 CONFIG_ELF_BUFFERINCR=32
 CONFIG_ELF_EXCLUDE_SYMBOLS=y
+CONFIG_ELF_CACHE_READ=y
```
Comment (sangam-swami) [inline build/configs/imxrt1050-evk/loadable_elf_apps/defconfig]: updated to enable cache read

File: build/configs/imxrt1050-evk/loadable_elf_apps/defconfig (hunk 1 above)
Comment (sangam-swami) [reply build/configs/imxrt1050-evk/loadable_elf_apps/defconfig]: enabled

**Comment by sangam-swami** [comment]
> could you let me know why this is needed?

For debugging and testing we need file system commands

**Pull request merged**